src/check.py

```diff
@@ -260,3 +265,60 @@ def check_heading_question_font(docx_file_path:str ,paragraphs:List[Paragraph]):
                 if "ＭＳ ゴシック" != content["font"] and "MS Gothic" != content["font"]:
                     return InvalidItem(type="フォント不正", message=f'「問~」のフォントがMSゴシックではありません')
                 buffer_question_no += content["text"]
+
+def check_exists_annotation(doc: Document):
+    """
+    傍注が本文内に正しく含まれているか検査する関数。
+
+    Parameters:
+        doc (Document): チェック対象のドキュメントオブジェクト。
+
+    Returns:
+        InvalidItem: 傍注が本文内に含まれていない場合のエラー情報。
+        None: 全ての傍注が本文内に正しく含まれている場合。
+    """
+
+    # 本文を抽出
+    main_texts = src.doc_util.extract_main_text(doc)
+
+    # 傍注リストを説明箇所から抽出
+    annotation_names = []
+    is_collecting_annotations = False
+    for paragraph in doc.paragraphs:
+        if paragraph.text.startswith("（注）"):
+            is_collecting_annotations = True
+        if is_collecting_annotations:
+            if paragraph.text.startswith("問"):
```

Comment:
is_collecting_annotationsというフラグの意味が良く解りませんでした

こうなるような
```suggestion
        if paragraph.text.startswith("（注）"):
            if paragraph.text.startswith("問"):
```